ios/kmbuild.sh

```diff
@@ -195,12 +203,6 @@ update_bundle ( ) {
       echo "${SUCCESS_GREEN}Packages successfully updated${NORMAL}"
 
       # If we aren't downloading resources, make sure copies of them already exist!
```

Comment:
This comment can be removed/moved

Reply:
Ack, forgot to delete this comment first; was trying not to overtax the CI at first.

Ah well, at least it's just a comment; won't break anything.